website/content/en/docs/building-operators/ansible/development-tips.md

```diff
@@ -31,7 +31,7 @@ sudo dnf install ansible
 ```
 
 In addition to Ansible, a user must install the
-[OpenShift Restclient Python][openshift-restclient-python] package:
+[Python Kubernetes Client][python-kubernetes-client] package:
 
 ```sh
 pip3 install openshift
```

Comment:
Does this need to be a different command or removed?

Reply:
@fabianvf I did a lazy-quick search, do you remember what openshift client was needed for?

Reply:
FWIW We are planning to remove it from the base container in a follow-up PR. We had some conversations with some folks working on Ansible that said all the functionality needed should be in the `kubernetes.core` collection now and that the openshift restclient is deprecated

Reply:
Oops, yeah, def missed that reference.